curweather/curweather.php

```diff
@@ -183,28 +207,38 @@ function curweather_addon_settings(&$a,&$s) {
 		'$curweather_units' => [ 'curweather_units', L10n::t('Units'), $curweather_units, L10n::t('select if the temperature should be displayed in &deg;C or &deg;F'), ['metric'=>'°C', 'imperial'=>'°F']],
 		'$enabled' => [ 'curweather_enable', L10n::t('Show weather data'), $enable, '']
 	]);
+
 	return;
 }
 
 // Config stuff for the admin panel to let the admin of the node set a APPID
 // for accessing the API of openweathermap
-function curweather_addon_admin_post (&$a) {
-	if(! is_site_admin())
+function curweather_addon_admin_post(App $a)
+{
+	if (! is_site_admin()) {
```

Comment:
Please remove the space after the ! operator.

Reply:
All done. And I found a few missing spaces more.